src/Sulu/Bundle/SecurityBundle/Entity/BaseRole.php

```diff
@@ -57,6 +57,20 @@
     private $securityType;
 
     /**
+     * Update the timestamps (changed + created).
+     * This method is mapped to the PRE_PERSIST and PRE_UPDATE
+     * lifecycle events.
+     */
+    public function updateTimestamps()
```

Comment:
As a side-note we would also need to add this to the mappings for Doctrine where applicable .. inheritance would simplify that but not sure if Doctrine mapping supports anything like "traits"